### PR TITLE
stylized dashboard and added shipping address edit function

### PR DESCRIPTION
**Pull request by ghost**
[trello card](https://trello.com/c/GIdYmqlE/100-as-a-signed-in-customer-i-will-see-all-details-about-my-account-on-my-customer-dashboard)

### CODE REVIEW

File: app/helpers/customers_helper.rb [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+      ['Kentucky', 'KY'],
+      ['Louisiana', 'LA'],
+      ['Maine', 'ME'],
+      ['Maryland', 'MD'],
+      ['Massachusetts', 'MA'],
+      ['Michigan', 'MI'],
+      ['Minnesota', 'MN'],
+      ['Mississippi', 'MS'],
+      ['Missouri', 'MO'],
+      ['Montana', 'MT'],
```
[the remark this answers is not in the document]
Comment (heydanhey) [reply app/helpers/customers_helper.rb]: I don't think it really matters here...but I guess it's considered the [ruby way](https://stackoverflow.com/questions/36162071/why-favor-the-use-of-w-over-foo-bar)

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,9 +5,7 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
-    <link href="https://fonts.googleapis.com/css?family=EB+Garamond|Lobster" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=EB+Garamond|Lobster|Roboto|Pangolin" rel="stylesheet">
```
Comment (heydanhey) [inline app/views/layouts/application.html.erb:8]: Nice!

File: app/views/customers/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<body class="botysbackground-dark">
+  <div class="grid">
+    <h4 class="center lobster-font beige underline">Edit Shipping Address</h4>
+    <br>
+    <%= form_tag "/customers/#{@customer.id}", method: :patch do %>
+
+      <div class="field">
+        <%= label_tag :first_name %><br />
+        <%= text_field_tag :first_name %>
```
Comment (heydanhey) [inline app/views/customers/edit.html.erb]: @charliedayman Great work! I was wondering if you could add the `@customer` info to the input field if it already exists? For example on line 9: `<%= text_field_tag :first_name, @customer.first_name %>` What do you think?

Comment (ghost) [reply app/views/customers/edit.html.erb]: Good idea @heydanhey! I'll add that in and push it up.

Comment (heydanhey) [reply app/views/customers/edit.html.erb]: Cool

**Review by heydanhey — APPROVED** [review]
+1, Thanks for making the changes - You're good to merge!!!

**Comment by heydanhey** [comment]
@charliedayman MERGE IT
MERGE

**Pull request merged**